src/common/modals/InstanceManager/Overview.js

```diff
@@ -175,6 +186,10 @@ const Overview = ({ instanceName, background, manifest }) => {
   const [javaLocalArguments, setJavaLocalArguments] = useState(
     config?.javaArgs
   );
+  const [customJavaPathSwitch, setCustomJavaPathSwitch] = useState(
+    config?.customJavaPath !== undefined
```

Comment:
why not `Boolean(config?.customJavaPath)`?

Reply:
Also what's the point on having this variable when you can just check `Boolean(config?.customJavaPath)` of `customJavaPath`?

Reply:
I did that, in the same way, it's done for JavaMemory